verticapy/tests/base.py

```diff
@@ -208,3 +209,27 @@ def require_DB_version_at_least(self, min_version):
         pass
 
     # Common assertions
+
+
+base_test = VerticaPyTestBase()
+base_test.setUp()
+
+if not (
+    os.path.isfile(os.path.dirname(verticapy.__file__) + "/tests/verticaPy_test.conf")
+):
+    path = os.path.dirname(verticapy.__file__) + "/tests/verticaPy_test.conf"
+    confparser = ConfigParser()
+    confparser.optionxform = str
+    try:
+        confparser.read(path)
```

Comment:
I don't see what is the point of this line.

Reply:
Corrected